qiskit/synthesis/permutation/permutation_lnn.py

```diff
@@ -20,11 +20,10 @@ def synth_permutation_depth_lnn_kms(pattern):
     """Synthesize a permutation circuit for a linear nearest-neighbor
     architecture using the Kutin, Moulton, Smithline method.
 
-    This is the permutation synthesis algorithm from
-    https://arxiv.org/abs/quant-ph/0701194, Chapter 6.
+    This is the permutation synthesis algorithm from [1], Chapter 6.
```

Comment:
I think it should be "Section 6" and not "Chapter 6" since it's a paper and not a book